tests/phpunit.xml

```diff
@@ -8,8 +8,5 @@
     <testsuite name="Twilio Integration Test Suite">
       <directory>./Twilio/Integration</directory>
     </testsuite>
-    <testsuite name="Twilio Services Test Suite">
```

Comment:
why was this removed?

Reply:
It doesn't actually exist in the repo. Removing phpdox from the dependencies list caused phpunit to use a newer version (since phpdox was lowering it before) which caused the tests to break because of this suite.